modules/app/routes/secure.js

```diff
@@ -18,7 +18,8 @@ module.exports = (req, res, next) => {
 
       req.decoded = decoded
       req.user = await userExists(decoded).catch(next)
-
+      // this will be the sender on emails
```

Comment:
@azdrenymeri why we need this? The `req.user` contains all information, anyway, the mail function is dispatched from an action on the from front-end, so we get the current logged user from there. Let me know where this is used.

Reply:
Sorry for my ignorance I totally forgot to change the code here.
`body.sender` supposed to be my quick test fix.
1. `body.sender` is destructured inside [taskMessage.js](https://github.com/worknenjoy/gitpay/pull/595/files#diff-089db36e510412c22df60b7fe19f3cfdR7) function.
2. Then is passed inside to  [AssignMail.messageInterested](https://github.com/worknenjoy/gitpay/pull/595/files#diff-089db36e510412c22df60b7fe19f3cfdR17)
```javascript
AssignMail.messageInterested(
        targetInterested.User.dataValues,
        task.dataValues,
        message,
        sender
      )
)
```